src/sonic-yang-mgmt/sonic_yang.py

```diff
@@ -50,7 +51,18 @@ def __init__(self, yang_dir, debug=False):
     def __del__(self):
         pass
 
+    def disable_logging(self):
+        ly.set_log_options(0)
+        self.syslogging_disabled = True
+
+    def enable_logging(self):
+        # ly.LY_LOLOG: If callback is set use it, otherwise just print. If flag is not set, do nothing.
+        ly.set_log_options(ly.LY_LOLOG)
+        self.syslogging_disabled = False
+
     def sysLog(self, debug=syslog.LOG_INFO, msg=None, doPrint=False):
+        if self.syslogging_disabled:
```

Comment:
>syslogging_disabled [](http://example.com/codeflow?start=16&length=19)

The name is confusing, it actually disabled syslog and print. Your intention is to only disable print?

Reply:
my intention is to disable logging

Reply:
I think it will be hard to read the logs as well if we keep printing errors during a change validation

Reply:
I see why it might be confusing, the name of the property `syslogging_disabled` but it is disabling it for both `syslog` and `print`. I got the name of the property from the name of the method `sysLog`.

Reply:
renamed to `logging_disabled`